source/docs/reference/cli.md

```diff
@@ -0,0 +1,93 @@
+---
+id: cli
+title: CLI
+---
+
+With Front-Commerce comes a CLI tool (`front-commerce`) that helps you launch the tasks needed to develop, build and launch your application.
+
+These commands should be launched from your project's root directory. This can be done by:
+* Using npm scripts (just like [front-commerce-skeleton/package.json](https://gitlab.com/front-commerce/front-commerce-skeleton/blob/master/package.json#L7))
+* Prefexing the commands with `npx` which will call the `front-commerce` bin in your project
+
+## `front-commerce help`
+
+Displays a message referencing the different commands available.
+
+## `front-commerce lint`
+
+This command ensures that your project is correctly linted. This means that it will:
+* format your code with prettier
+* prevent errors by using eslint rules
+
+In order to configure the rules, please refer to the `eslintConfig` key in your package.json.
+
+A basic config could be :
+
+```
+{
+  "extends": [
+    "eslint:recommended",
+    "react-app"
+  ],
+  "plugins": [
+    "prettier",
+    "security"
+  ],
+  "rules": {
+    "no-console": [
+      "warn",
+      {
+        "allow": [
+          "warn",
+          "error"
+        ]
+      }
+    ],
+    "prettier/prettier": "warn"
+  }
+}
+```
+
+The files linted are the files within the modules you have defined in [`.front-commerce.js`](/docs/reference/front-commerce-js.html). However, the modules that are in `node_modules` won't be linted.
+
+> Please note that the errors and warning that this tool will log are also displayed when using `front-commerce start` in development mode. The `front-commerce lint` command is mainly useful when you want to ensure that your code is correctly linted before commiting it to your repository.
+
+## `front-commerce prepare`
+
+This command will generate all the files needed to launch a Front-Commerce application. It will use the [`.front-commerce.js`](/docs/reference/front-commerce-js.html) to know what to generate. 
```

Comment:
```suggestion
This command will generate all the files needed to launch a Front-Commerce application. It will use the [`.front-commerce.js`](/docs/reference/front-commerce-js.html) configuration file to know what to generate. 
```

I think it is clearer this way, to ensure people clearly separate the `.front-commerce` directory from the config file.